### PR TITLE
Revise steps for clarity and dist UI components, bootable-usb.

### LINKED ISSUE #1033: TCS editorial review of bootable-usb.rst
Apply editorial review for BalenaEtcher / bootable-usb, which was merged on 02-19-20. 
- Correct usage of `balenaEtcher` to `Etcher` per USPTO. 
- Integrate an Advanced section per CL Docs discussion/agreement
- Test, verify, and correct output in Advanced section

### PR DESCRIPTION
**Pull request by mvincerx**
- Resize all images for uniformity
- Revise from balenaEtcher to Etcher per USPTO trademark
- Closes #1033 
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by mvincerx** [comment]
@odwdinc, please review. 
@bktan8, please review Advanced section--in honor of your work for Linux CLI.

**Comment by mvincerx** [comment]
@odwdinc Will you have time to review this today? 
I'm a little concerned on timing. Please see my email.

File: source/get-started/bootable-usb.rst [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 *************
 
 * Download the |CL| Desktop or Server image from the `Downloads`_ page
 * Recommended minimum **4GB** USB drive or larger
-* Download and install `BalenaEtcher`_
-
-.. _bootable-usb-linux:
+* Download and install the `Etcher`_ version per your operating system.
 
 * Make sure you completed all `Prerequisites`_.
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: Seems redundant.

File: source/get-started/bootable-usb.rst [hunk 2]
```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
...
    .. figure:: /_figures/bootable-usb/balenaEtcher_ReadyToFlash.PNG
       :scale: 100%
-      :alt: 
+      :alt: USB Flash Device selected
       
-      Figure X: 
+      Figure 4: USB Flash Device selected
 
-#. When ready Press the  :guilabel:`Flash!` Button.
+#. When ready Press the :guilabel:`Flash!` Button.
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: "Press" should be "press" (lowercase)

File: source/get-started/bootable-usb.rst [hunk 3]
```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
...
 
-#. Flashing starting.
+#. The dialog shows :guilabel:`Flashing` while in progress.
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: Steps 9-11 should not be steps IMO.  These are not actions for the user to take, so they shouldn't be number.  The should be just information for the previous step.

File: source/get-started/bootable-usb.rst [hunk 4]
```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
...
+      sudo umount /dev/sdc*   
+
+#. Burn the image onto the USB drive. This example burns an image onto
+   ``/dev/sdc``. The device name of the USB may vary.   
+
+   .. code-block:: bash
+
+      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdc oflag=sync bs=4M status=progress
 
 Ejecting the |CL| image USB drive
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]:
```suggestion
Eject the |CL| image USB drive
```

File: source/get-started/bootable-usb.rst [hunk 5]
```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
    Burning an image formats the USB drive and destroys all pre-existing
    content.  Back up your data before proceeding.
 
-#. Launch balenaEtcher.
+#. Launch Etcher.
 
    .. figure:: /_figures/bootable-usb/balenaEtcher_Start.PNG
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: Please add drop shadows to all images.

File: source/get-started/bootable-usb.rst [hunk 6]
```diff
@@ -22,90 +23,140 @@ Burn the |CL| image onto a USB drive
...
 
    .. note::
-      The process can take more than a few minutes. When the process completes, close BalenaEtcher.
+
+      The process may take more than a few minutes. When the process completes, close BalenaEtcher.
       
+Advanced: Linux CLI
+===================
+
+#. Open a Terminal window
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]:
```suggestion
#. Open a Terminal window.
```

File: source/get-started/bootable-usb.rst (hunk 1 above)
Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: Agreed. I'll omit that final line.

File: source/get-started/bootable-usb.rst (hunk 5 above)
Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: Sure. Sounds good.

File: source/get-started/bootable-usb.rst (hunk 2 above)
Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: 👍

File: source/get-started/bootable-usb.rst (hunk 3 above)
Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: I agree. Good suggestion.

File: source/get-started/bootable-usb.rst (hunk 4 above)
Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: Done.

**Comment by mvincerx** [comment]
@bktan8. Please check out my last push in response to your feedback. 😄

**Pull request merged**